src/pidNames.jl

```diff
@@ -0,0 +1,612 @@
+PIDNames = Dict(
+    1 => "d",
+    -1 => "d̄", 
+    2 => "u",
+    -2 => "ū", 
+    3 => "s",
+    -3 => "s̄", 
+    4 => "c",
+    -4 => "c̄", 
+    5 => "b",
+    -5 => "b̄", 
+    6 => "t",
+    -6 => "t̄",
+    11 => "e⁻",
+    -11 => "e⁺",
+    12 => "νₑ",
+    -12 => "ν̄ₑ",
+    13 => "μ⁻",
+    -13 => "μ⁺",
+    14 => "ν(μ)",
+    -14 => "ν̄(μ)",
+    15 => "τ⁻",
+    -15 => "τ⁺",
+    16 => "ν(τ)",
+    -16 => "ν̄(τ)",
+    17 => "τ'",
+    -17 => "τ'",
+    18 => "ν(τ')",
+    -18 => "ν̄(τ')",
+    21 => "g",
+    22 => "γ",
+    23 => "Z⁰",
+    24 => "W⁺",
+    -24 => "W⁻",
+    25 => "H⁰",
+    111 => "π⁰",
+    113 => "ρ(770)⁰",
+    115 => "a₂(1320)⁰",
+    117 => "ρ₃(1690)⁰",
+    119 => "a₄(1970)⁰",
+    130 => "K(L)",
+    211 => "π⁺",
+    -211 => "π⁻",
+    213 => "ρ(770)⁺",
+    -213 => "ρ(770)⁻",
+    215 => "a₂(1320)⁺",
+    -215 => "a₂(1320)⁻",
+    217 => "ρ₃(1690)⁺",
+    -217 => "ρ₃(1690)⁻",
+    219 => "a₄(1970)⁺",
+    -219 => "a₄(1970)⁻",
+    221 => "η",
+    223 => "ω(782)",
+    225 => "f₂(1270)",
+    227 => "ω₃(1670)",
+    229 => "f₄(2050)",
+    310 => "K(S)",
+    311 => "K⁰",
+    -311 => "K̄⁰", 
+    313 => "K*(892)",
+    -313 => "K̄*(892)",
+    315 => "K(2)*(1430)",
+    -315 => "K̄(2)*(1430)",
+    317 => "K(3)*(1780)",
+    -317 => "K̄(3)*(1780)",
+    319 => "K(4)*(2045)",
+    -319 => "K̄(4)*(2045)",
+    321 => "K⁺",
+    -321 => "K⁻",
+    323 => "K*(892)",
+    -323 => "K̄*(892)",
+    325 => "K(2)*(1430)",
+    -325 => "K̄(2)*(1430)",
+    327 => "K(3)*(1780)",
+    -327 => "K̄(3)*(1780)",
+    329 => "K(4)*(2045)",
+    -329 => "K̄(4)*(2045)",
+    331 => "eta'(958)",
+    333 => "ϕ(1020)",
+    335 => "f(2)'(1525)",
+    337 => "ϕ₃(1850)",
+    411 => "D⁺",
+    -411 => "D⁻",
+    413 => "D*(2010)",
+    -413 => "D̄*(2010)",
+    415 => "D(2)*(2460)",
+    -415 => "D̄(2)*(2460)",
+    421 => "D⁰",
+    -421 => "D̄⁰", 
+    423 => "D*(2007)",
+    -423 => "D̄*(2007)",
+    425 => "D(2)*(2460)",
+    -425 => "D̄(2)*(2460)",
+    431 => "Dₛ⁺",
+    -431 => "Dₛ⁻",
+    433 => "D(s)*",
+    -433 => "D̄(s)*",
+    435 => "D(s2)*(2573)",
+    -435 => "D̄(s2)*(2573)",
+    441 => "eta(c)(1S)",
+    443 => "J/ψ(1S)",
+    445 => "χ(c2)(1P)",
+    511 => "B⁰",
+    -511 => "B̄⁰", 
+    513 => "B*",
+    -513 => "B*",
+    515 => "B(2)*(5747)",
+    -515 => "B̄(2)*(5747)",
+    521 => "B⁺",
+    -521 => "B⁻",
+    523 => "B*",
+    -523 => "B̄*",
+    525 => "B(2)*(5747)",
+    -525 => "B̄(2)*(5747)",
+    531 => "Bₛ⁰",
```

Comment:
some of the characters like this are displayed es empty boxes on my machine 😕

Reply:
Should we have the names in a hardcoded dictionary or or also store them in some kind of catalog file, e.g. csv. Then users can create a custom file without changing the code itself if those problems occur.

Reply:
That's what Jan also proposed before. Yeah, maybe that's the way to go, now that I see that huge list.
I think in this case we should create a workflow to generate that from the particle table, so that the only manual step is -- whenever there is an update on the particle tables from the SciKit-HEP guys - to copy the new files, run the script and we are good to go.

Reply:
I think we don't have to break this workflow, because the names don't need to be updated for every new particle table. Only when new particles are present ... for example if we also introduce nuclei :wink:

Reply:
But the names "are" in a hard-coded dictionary. I'm not sure what you are proposing to change.
Are you suggesting to write the dictionary to a `csv` file? That's easily done, but I don't see what the advantage is over changing the `.jl` file.

Reply:
> I think we don't have to break this workflow, because the names don't need to be updated for every new particle table. Only when new particles are present ... for example if we also introduce nuclei

Right. My thinking, too. I think our posts crossed...

Reply:
I guess the main intention was that the user could basically provide their own `CSV` file and then `load_aliases()` or whatever, but now I also think that's a bit of an overkill 😉

Reply:
Yeah, that's what I meant just moving it to a `csv`. This is more a structural thing, because so far we have all data for conversions and lookups in `data` and I feel not so well with such a big dictionary in a `jl` file... but I guess me feeling not well can be quickly overruled by coding facts :wink: